tests/test_date_filter.py

```diff
@@ -1,6 +1,8 @@
 from dateutil.parser import parse
 import json
 
+import mock
```

Comment:
While `mock` is part of the standard lib in Python 3.3+, prefer the backport package in order to support Python 2.7.